notebooks/en/_toctree.yml

```
@@ -2,37 +2,47 @@
   sections:
   - local: index
     title: Open-Source AI Cookbook
-  - local: issues_in_text_dataset
-    title: Detecting Issues in a Text Dataset with Cleanlab
-  - local: stable_diffusion_interpolation
-    title: Stable Diffusion Interpolation
-  - local: rag_with_hugging_face_gemma_mongodb
-    title: Building A RAG System with Gemma, MongoDB and Open Source Models
-  - local: tgi_messages_api_demo
-    title: Migrating from OpenAI to Open LLMs Using TGI's Messages API
+    
+- title: LLM Recipes
+  sections:
   - local: automatic_embedding_tei_inference_endpoints
     title: Automatic Embeddings with TEI through Inference Endpoints
-  - local: faiss_with_hf_datasets_and_clip
-    title: Embedding multimodal data for similarity search
+  - local: tgi_messages_api_demo
+    title: Migrating from OpenAI to Open LLMs Using TGI's Messages API
+  - local: advanced_rag
+    title: Advanced RAG on HuggingFace documentation using LangChain
+  - local: labelling_feedback_setfit
+    title: Suggestions for Data Annotation with SetFit in Zero-shot Text Classification
   - local: fine_tuning_code_llm_on_single_gpu
     title: Fine-tuning a Code LLM on Custom Code on a single GPU
+  - local: prompt_tuning_peft
+    title: Prompt tuning with PEFT
+  - local: rag_evaluation
+    title: RAG Evaluation
+  - local: llm_judge
+    title: Using LLM-as-a-judge for an automated and versatile evaluation
+
+- title: Diffusion Recipes
+  sections:
+  - local: stable_diffusion_interpolation
+    title: Stable Diffusion Interpolation
+
+- title: Multimodal Recipes
+  - local: faiss_with_hf_datasets_and_clip
```

The missing `sections` is causing the doc-builder to fail

```suggestion
- title: Multimodal Recipes
  sections:
  - local: faiss_with_hf_datasets_and_clip
```